src/repositories/LikeRepository.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-useless-catch */
+import database from '../models';
+
+const { Like, Accommodation } = database;
+
+class LikeRepository {
+  /**
+   * Creates a new Like.
+   * @param {object} user_id
+   * @param {object} accommodation_id
+   * @returns {object} Like.
+   */
+  static async Like({ user_id, accommodation_id }) {
+    try {
+      return await Like.create({
+        user_id,
+        accommodation_id
+      }, {
+        fields: ['user_id', 'accommodation_id']
+      });
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  /**
+   * check if accommodation exists.
+   * @param {object} accommodation_id .
+   * @returns {object} Like object.
+   */
+  static async checkAccommodation(accommodation_id) {
+    try {
+      return await Accommodation.findOne({
```

Comment:
Same here, remove the `await` keyword or assign to a variable that you'll be returning.